Tryouts/Prototypes/Shell/Manifest/ManifestParser.cs

```diff
@@ -20,26 +20,28 @@
 using System.Text.Json;
 using System.Windows.Automation;
 using System.Windows.Controls.Primitives;
+using System.Windows.Documents;
 
 namespace Manifest
 {
-    internal class ManifestParser
+    internal static class  ManifestParser
     {
-        internal ManifestModel Manifest { get; set; }
+        internal static ManifestModel Manifest { get; set; }
 
-        public ManifestParser()
+        public static  ManifestModel OpenManifestFile(string fileName)
```

Comment:
`OpenManifestFile` suggests it returns something that can be later closed. Just call it `Parse` or `ParseManifest`.